packages/stentor-guards/src/isHandler.ts

```diff
@@ -6,7 +6,7 @@ import { Handler, Intent } from "stentor-models";
  *
  * @public
  */
-export function isHandler(props: Handler | Intent): props is Handler {
+export function isHandler(props: Handler | Intent | object | number | boolean): props is Handler {
```

Comment:
How can a number be a handler candidate? And what object if not handler or intent?

Reply:
Really never, I only added those to make it easier to in cases where a Handler is a potential input to a function and to appease TypeScript, I add those to simplify the logic needed.  See [here](https://github.com/stentorium/stentor/pull/518/files#diff-a5fd25c20dd0ac3db6349d85b9fe7cb4R30) where `isHandler(content)` and `content` can potentially be another `object` or an array (also typed as an `object`).  I don't have to add those extra checks before calling `isHandler`